Automattic/jetpack
# CRM Automations: Add Bulk Workflow Actions Component

Fixes https://github.com/Automattic/zero-bs-crm/issues/3209

## Proposed changes:


This PR adds the Bulk Workflow Actions components.

### Other information:


## Jetpack product discussion


NtYgueI8oAgiWGHTbHpnjh-fi-1_29

## Does this pull request change what data or activity we track or use?



## Testing instructions:






1. Review the code.
2. Put `add_filter( 'jetpack_crm_feature_flag_automations', '__return_true' );` in an mu-plugin.
3. Visit `/wp-admin/admin.php?page=jpcrm-automations#/automations/test-workflow-bulk-actions`. Check that the "bulk actions" dropdown shows and matches the visuals in the Figma. Note that no functionality is implemented yet. This is waiting on the store implementation.
